config/jobs/kubeflow/kubeflow-presubmits.yaml

```diff
@@ -199,33 +199,33 @@ presubmits:
       - image: gcr.io/kubeflow-ci/test-worker:latest
         imagePullPolicy: Always
 
+
   kubeflow/pipelines:
-  - name: kubeflow-pipeline-presubmit
+  - name: kubeflow-pipeline-e2e-test
     always_run: true
     decorate: true
     labels:
       preset-service-account: "true"
-    agent: knative-build
-    build_spec:
-      steps:
-      - name: kubeflow-pipeline-build-image
-        image: gcr.io/k8s-testimages/kubekins-e2e:v20190205-d83780367-1.10
-        command:
-        - ./test/presubmit-tests.sh
-        args:
-        - --workflow_file
-        - build_image.yaml
-      - name: kubeflow-pipeline-e2e-test
-        image: gcr.io/k8s-testimages/kubekins-e2e:v20190205-d83780367-1.10
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190205-d83780367-1.10
         command:
         - ./test/presubmit-tests-with-pipeline-deployment.sh
         args:
         - --workflow_file
         - e2e_test_gke_v2.yaml
         - --test_result_folder
         - e2e_test
-      - name: kubeflow-pipeline-sample-test
-        image: gcr.io/k8s-testimages/kubekins-e2e:v20190205-d83780367-1.10
+
+  kubeflow/pipelines:
```

Comment:
drop the extra `kubeflow/pipelines` key?

Reply:
done